lib/FrontendTool/FrontendTool.cpp

```diff
@@ -314,47 +311,43 @@ static bool writeSIL(SILModule &SM, const PrimarySpecificPaths &PSPs,
                   PSPs.OutputFilename, opts.EmitSortedSIL);
 }
 
-static bool printAsObjCIfNeeded(StringRef outputPath, ModuleDecl *M,
-                                StringRef bridgingHeader, bool moduleIsPublic) {
-  using namespace llvm::sys;
-
+/// A wrapper around swift::atomicallyWritingToFile that handles diagnosing any
+/// filesystem errors and ignores empty output paths.
```

Comment:
Suggest you add a Doxygen comment about the return convention, since we don't use enums.
Returns true on error?